src/main/java/com/example/demo/OrderComponent/API/DTOs/ModifyOrderDTO.java

```diff
@@ -0,0 +1,20 @@
+package com.example.demo.OrderComponent.API.DTOs;
+
+import com.example.demo.OrderComponent.Domain.Entities.Enums.OrderStatus;
+import lombok.*;
+
+import java.util.List;
+import java.util.UUID;
+
+@Getter
+@Setter
+@NonNull
```

Comment:
NonNull/NotNull usually goes on the parameters of a function, the fields themselves, not the whole class